docker-compose.yml

```diff
@@ -2,8 +2,22 @@ version: "3"
 
 services:
   medmorph_ehr:
-    image: medmorph_ehr
+    image: medmorph_fhir
     container_name: medmorph_ehr
     restart: on-failure
     ports:
       - "8080:8080"
+    environment:
+      - SERVER_ADDRESS=http://localhost:8080/fhir/
+      - AUTH_SERVER_ADDRESS=http://moonshot-dev.mitre.org:8090/auth/realms/ehr/protocol/openid-connect/
+      - SERVER_TITLE='MedMorph EHR'
+  knowledge_artifact:
+    image: medmorph_fhir
+    container_name: knowledge_artifact
+    restart: on-failure
+    ports:
+      - "8090:8080"
+    environment:
+      - SERVER_ADDRESS=http://localhost:8090/fhir/
+      - AUTH_SERVER_ADDRESS=http://moonshot-dev.mitre.org:8090/auth/realms/knowledgeartifact/protocol/openid-connect/
+      - SERVER_TITLE='Knowledge Artifact Repository'
```

Comment:
Super super minor thing I noticed in the demo, but the name here seems to be double quoted in actual usage. (Which is strange, I thought you didn't need quotes in YML but putting them in anyway wouldn't break anything)
From http://pathways.mitre.org:8190/fhir/metadata:

```
{
  "resourceType": "CapabilityStatement",
  "title": "'Knowledge Artifact Repository'",
  "status": "active",
  "experimental": true,
  "date": "2021-01-21T14:45:28+00:00",
  ```

Reply:
So it turns out unlike environment variables in bash you can have spaces and don't need to wrap it in quotes at all